src/pages/staking-hub/onboarding/step2/0fundSafe.tsx

```diff
@@ -91,6 +111,22 @@ const FundsToSafe = () => {
     write_wxHOPR_to_safe?.();
   };
 
+  const xdaiEnoughBalance = (): boolean => {
+    console.log(safeBalance.xDai.value);
+    if (Number(safeBalance.xDai.value) >= 0.1) {
```

Comment:
value"200000000000000"
formatted:"0.0002"
gives `enough balance` although shouldn't

Reply:
Good catch! This is fixed now